src/components/organisms/LoginForm.tsx

```diff
@@ -0,0 +1,82 @@
+import { AuthService } from '@src/services';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Space } from '../atoms';
+export default function LoginForm() {
+  const [loginFormState, setLoginFormState] = useState({
+    email: '',
+    password: '',
+  });
+
+  const handleLoginFormChange = (e) => {
+    if (/[ㄱ-ㅎ|ㅏ-ㅣ|가-힣]/.test(e.target.value)) {
+      alert('한글 입력은 지원하지 않습니다. 영문으로 전환해주세요.');
+      focus;
+      return;
+    }
+    setLoginFormState({ ...loginFormState, [e.target.name]: e.target.value });
+  };
+
+  const handleLogin = async () => {
+    if (loginFormState.email === '' || loginFormState.password === '') {
+      alert('아이디/비밀번호를 확인해주세요');
+    } else {
+      await AuthService.superToken(
+        loginFormState.email,
+        loginFormState.password
+      );
+      alert('로그인 성공');
+    }
```

Comment:
```suggestion
// 함수 외부
const isValidForm = loginFormState.email !== '' || loginFormState.password !== '';
...

// 함수 내부
    if (!isValidForm) {
      alert('아이디/비밀번호를 확인해주세요');
      return;
    }
    
    try { 
      await AuthService.superToken(
        loginFormState.email,
        loginFormState.password
      );
      alert('로그인 성공');
    } catch (e) {
      alert('로그인에 실패했습니다' + e);
    }
```

1. 함수를 분리해서 각 함수의 몸통을 줄여보세요.
2. escape case를 다룰땐 return 시켜버리기, 되도록 함수 몸통의 맨 윗부분에 작성하기
3. try-catch 잘 활용하기

Reply:
8f0bd28